pkg/secrets/provide_conjur_secrets_test.go

```diff
@@ -14,11 +14,31 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	providerDelayTime = 500
```

Comment:
I think we can get by with 10s of milliseconds instead of 100s.
For example, this seems to work:
https://github.com/cyberark/secrets-provider-for-k8s/commit/d61c96c091f171be0aa04be5c75a6407a3e5f092#diff-c0314624500148cb0ccac2409ef0d4dc9a8cddf7603e8879bae326cc2d96d7d5L132-R176

Reply:
made the change.